infra/bicep/main.bicep

```diff
@@ -73,6 +73,17 @@ param existing_AppInsights_Name string = ''
 // --------------------------------------------------------------------------------------------------------------
 @description('If you provide this is will be used instead of creating a new Container App Environment')
 param existing_managedAppEnv_Name string = ''
+@description('Name of the Container Apps Environment workload profile to use for the app')
+param appContainerAppEnvironmentWorkloadProfileName string = 'app'
+@description('Workload profiles for the Container Apps environment')
+param containerAppEnvironmentWorkloadProfiles array = [
+  {
+    name: 'app'
+    workloadProfileType: 'D4'
```

Comment:
the apps are pretty lightweight - do we need D4 workload profile?

It would be good to switch to workload profile consumption since it's supports private endpoints

Reply:
Was just trying to match what Scott has in his UI sample - because I kept having issues and wasn't sure why...
now I know if was the NSG